external/emulation/emulation/_monitor/monitor.py

```diff
@@ -26,18 +27,29 @@
 }
 
 
-def _bool_from_str(value: str):
-    affirmatives = ["y", "yes", "true"]
-    negatives = ["n", "no", "false"]
+@dataclasses.dataclass
+class StorageConfig:
+    """Storage configuration
+
+    Attributes:
+        output_freq_sec: output frequency in seconds to save
+            nc and/or zarr files at
+        var_meta_path: path to variable metadata added to saved field
+            attributes. if not set, then the environmental variable VAR_META_PATH
+            will be used. If THAT isn't set then no metadata other than 'unknown'
+            units will be set.
+        save_nc: save all state fields to netcdf, default
+            is true
+        save_zarr: save all statefields to zarr, default
+            is true
+        save_tfrecord: save all statefields to tfrecord
+    """
 
-    if value.lower() in affirmatives:
-        return True
-    elif value.lower() in negatives:
-        return False
-    else:
-        raise ValueError(
-            f"Unrecognized value encountered in boolean conversion: {value}"
-        )
+    var_meta_path: str = ""
+    output_freq_sec: int = 1
```

Comment:
1 s seems pretty high frequency for a default, maybe keep the old default which saved every 10800 s.